libs/tokens/src/index.ts

```diff
@@ -0,0 +1,43 @@
+// Updaters
+export { TokensListsUpdater } from './updaters/TokensListsUpdater'
+
+// Pure components
+export { TokenLogo } from './pure/TokenLogo'
+
+// Types
+export * from './types'
+export type { TokensByAddress, TokensBySymbol } from './state/tokens/allTokensAtom'
```

Comment:
nit: I would say, these types belong to th `types.ts` if they are part of the interface.
Then, it just happens that you actually also use them in the atom because in this case you made the persistance to match the interface